.github/workflows/kind-e2e.yaml

```diff
@@ -20,9 +20,9 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.23.12
-        - v1.24.6
-        - v1.25.2
+        - v1.23.13
```

Comment:
Could you drop 1.23.x ?
It is no longer supported as https://github.com/knative/pkg/pull/2639 

The CI on this PR https://github.com/knative-sandbox/net-istio/pull/1009 could be fixed once we dropped 1.23.

Reply:
done.

thanks for the pointing out the knative/pkg no longer support 1.23.x